src/transforms/globalCssToCssModule/ts/processNodesWithClassName.ts

```diff
@@ -0,0 +1,42 @@
+import { ts, Identifier, StringLiteral } from 'ts-morph'
+
+import { getClassNameNodeReplacement } from './getClassNameNodeReplacement'
+import { splitClassName } from './splitClassName'
+
+export const STYLES_IDENTIFIER = 'styles'
+
+interface ProcessStringLiteralsOptions {
+    nodesWithClassName: (Identifier | StringLiteral)[]
+    exportNameMap: Record<string, string>
+}
+
+// Walk through all the `className` nodes and replace relevant classes with matching CSS module export names.
+export function processNodesWithClassName(options: ProcessStringLiteralsOptions): void {
```

Comment:
It might be useful to have a jsdoc comment with examples here like `splitClassName` function has